classes/image_layer.py

```diff
@@ -86,6 +87,7 @@ def import_str(self, import_str):
         self.__import_str = import_str
 
     def add_package(self, package):
+        assert isinstance(package, Package)
```

Comment:
AssertionError is blank here. If you can add an informational message about the assertion error, that would be great:
```
try:                                                                    
    assert isinstance(package, Package), "Object is not a Package"      
except AssertionError:
    raise 
```
```
>>> from classes.image_layer import ImageLayer
>>> i = ImageLayer('123abc')
>>> i.add_package('something')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/nisha/tern3.7/tern/classes/image_layer.py", line 91, in add_package
    assert isinstance(package, Package), "Object is not a Package"
AssertionError: Object is not a Package
```
Please make the error message more verbose: "Object is of type Package. See classes/package.py"